shell/browser/extensions/electron_extension_loader.cc

```diff
@@ -110,6 +110,9 @@ void ElectronExtensionLoader::FinishExtensionLoad(
   if (extension) {
     extension_registrar_.AddExtension(extension);
   }
+  ExtensionPrefs* extension_prefs = ExtensionPrefs::Get(browser_context_);
+  extension_prefs->OnExtensionInstalled(extension.get(), Extension::ENABLED,
```

Comment:
This addition comes with it persistence of a number of properties for each extension installed.

![image](https://user-images.githubusercontent.com/1656324/94649292-5d782d00-02c2-11eb-9ea6-bf5157fc6c79.png)

This goes back to some discussion in https://github.com/electron/electron/pull/25198#discussion_r487389995 . I can think of a few ways around this:
- Manually write the `install_time` property to avoid the DCHECK
- Patch //extensions to remove the DCHECK
- Switch to using the `InMemoryPrefStore` to avoid any of this from getting written to disk

I'll have to think on this more to see if there are other alternatives.

Reply:
This is still persisted to the `Preferences` JSON store, but I modified the code to write only the `install_time` pref in 
8d837e4

![image](https://user-images.githubusercontent.com/1656324/94655327-653ccf00-02cc-11eb-9a1a-7fb140bb04b2.png)